### ISSUE TITLE
Update `publish_docs_to_wiki` workflow to use `publish_wiki` of github_action_workflow repository

### ISSUE THREAD

**Issue by manh-t** [nimblehq/android-templates#230]
## Why

We have a `github_action_workflow` repository to maintain the reusable workflows, including [publish_wiki.yml](https://github.com/nimblehq/github-actions-workflows/blob/develop/.github/workflows/publish_wiki.yml) workflow. It's better not to fragment the workflows and maintain it ourself.

Updating `publish_docs_to_wiki` to extend and use the actions in [publish_wiki.yml](https://github.com/nimblehq/github-actions-workflows/blob/develop/.github/workflows/publish_wiki.yml).

## Who Benefits?

Developers.

**Comment by Tuubz** [comment]
@nimblehq/android-chapter Could I get more votes on this one, please? 🙏

**Comment by Tuubz** [comment]
Please check out [this issue](https://github.com/nimblehq/compass/discussions/1019) too, we might end up using [this repository](https://github.com/nimblehq/publish-github-wiki-action) instead 🙏

**Comment by manh-t** [comment]
@Tuubz yes, I'm aware of that issue. We might need to update this issue's description when the RFC is approved 🙏

**Comment by hoangnguyen92dn** [comment]
@Tuubz @manh-t To conclude, which workflow we could use for now? `Updating the description when the RFC is approved` doesn't make sense imo.

**Comment by Tuubz** [comment]
@manh-t 

If you agree, please update the description to make use of [this repository](https://github.com/nimblehq/publish-github-wiki-action) 🚀
